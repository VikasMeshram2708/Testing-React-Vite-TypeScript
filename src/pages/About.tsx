export default function About() {
  return (
    <section data-testId="aboutPage">
      <h1>About Page</h1>
      <p>Lorem ipsum dolor sit amet.</p>
    </section>
  );
}
