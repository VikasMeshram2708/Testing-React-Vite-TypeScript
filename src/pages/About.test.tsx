import { render, screen } from "@testing-library/react";
import About from "./About";

describe("testing about page", () => {
  it("check the heading", () => {
    render(<About />);
    // const heading = screen.getByRole("heading");
    const AboutPage = screen.getByTestId("aboutPage");
    expect(AboutPage).toMatchSnapshot();
  });
});
