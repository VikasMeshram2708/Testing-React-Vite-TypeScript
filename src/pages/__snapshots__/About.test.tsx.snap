// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing about page check the heading 1`] = `
<section
  data-testid="aboutPage"
>
  <h1>
    About Page
  </h1>
  <p>
    Lorem ipsum dolor sit amet.
  </p>
</section>
`;
